test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -9,31 +10,42 @@ import {
   fundLnd,
   checkIsBalanced,
   getUserWallet,
+  mineAndConfirm,
+  sendToAddressAndConfirm,
   waitUntilBlockHeight,
 } from "test/helpers"
 
 jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
 
-const defaultWallet = ""
+let bitcoindOutside
+
+afterAll(async () => {
+  await bitcoindClient.unloadWallet({ wallet_name: "outside" })
+})
 
 describe("Bitcoind", () => {
-  it("create default wallet", async () => {
-    try {
-      const { name } = await bitcoindClient.createWallet(defaultWallet)
-      // depends of bitcoind version. needed in < 0.20 but failed in 0.21?
-      expect(name).toBe(defaultWallet)
-    } catch (error) {
-      baseLogger.warn({ error }, "bitcoind wallet already exists")
-    }
+  it("check no wallet", async () => {
     const wallets = await bitcoindClient.listWallets()
-    expect(wallets).toContain("")
+    expect(wallets.length).toBe(0)
+  })
+
+  it("create outside wallet", async () => {
```

Comment:
This doesn't have an `expect` so it should probably be in a `before` hook or somehow part of test setup code.